symfony/README.md

```diff
@@ -5,7 +5,7 @@ Algolia Symfony bundle was created by, and is maintained by [Algolia](https://gi
 ## Get started locally
 > requires :
 * [Composer](https://getcomposer.org/)
-* [PHP 7.1.3+](https://www.php.net/releases/index.php)
+* [PHP 7.4+](https://www.php.net/releases/index.php)
```

Comment:
The search bundle actually [supports 7.2 and up](https://github.com/algolia/search-bundle/blob/master/composer.json#L13). Would it be possible for us to update the `composer.lock` file while on PHP 7.2 (in the hopes it's compatible with later versions too)? Otherwise, we might want to remove the `composer.lock` file and have it generated whenever the user sets things up on their machine, based on their PHP version. WDYT?

```suggestion
* [PHP 7.2+](https://www.php.net/releases/index.php)
```

Reply:
I'll test that on the other branch, but as PHP 7.2 went EOL 1 year ago and PHP 7.3 will go in a few days, I think I'd rather bump the minimum supported version of the bundle: https://www.php.net/supported-versions.php

Reply:
Ok we can see later to bump the minimum version of everything, for now requiring PHP `7.2` and simply running `composer update` regenerated the lock file and fixed the issue. I'm now able to run the project with a 7.3 version.

I've also added `doctrineSubscribedEvents: []` in the config because I was surprised at first to see that a `php bin/console doctrine:fixtures:load` was also pushing records into Algolia.